doc/docs.md

```diff
@@ -2130,6 +2103,69 @@ In this example, the `can_register` method has a receiver of type `User` named `
 The convention is not to use receiver names like `self` or `this`,
 but a short, preferably one letter long, name.
 
+### Embedded structs
+
+V supports embedded structs , similar to inherit, parent-subclasses in other programming languages.
```

Comment:
It is not similar to inheritance, and the word inheritance is misleading and should not be used.
You are free to feel that way, but it shouldn't be in the documentation.

There is no need to compare it to anything.
But if I had to make an analogy, it would be delegation, not inheritance or parent-child relationships.
It would be more appropriate to compare it to delegation, where the compiler automatically provides wrapper methods for you.

Translated with www.DeepL.com/Translator (free version)

Reply:
@yuyi98  any comment about this?

Reply:
In the new version , I write like this

Embedded struct like inherit , struct will automatically have the fields and functions from embedded struct.

Embedded struct is not inherit, there is no parent-child relationship between embedded structs, and can't type cast.

Reply:
I like the new formulation 👍🏻 .

Reply:
except `, and can't type cast.` - that can be just removed

Reply:
oh, sorry - you already changed that 😄 .